Update image used for presto container

MasterOdin opened the pull request:
PR updates the source used for the `prestodb-sandbox` image, as the existing one was deleted from Docker Hub. Earlier this year IBM acquired Ahana, and last week it looks like they completed deleted all resources that Ahana had on a number of sites, including Docker Hub.

The image replacing it with is the exact same image, just uploaded to a new place. In theory, at some point it should be possible to use the `prestodb/presto` images, but currently using them requires creating and mounting a number of `etc/` config files, vs just having the image be self-contained.

tagomoris commented:
I'm receiving notifications of changes, but now can't do any work because of a disaster of flu&covid-19 in my home. Let me check them days or a week later.

MasterOdin commented on docker-compose.yml:
```diff
@@ -2,9 +2,7 @@ version: "3.9"
   presto:
-    image: ahanaio/prestodb-sandbox:0.281
-    # there is no arm64 version of this image
-    platform: linux/amd64
```
Can remove the `platform` flag here as our fork publishes both `linux/amd64` and `linux/arm64` for each version tag.

MasterOdin replied:
Turns out `linux/amd64` is still required. Not sure exactly what configuration I had tested to believe it worked on `linux/arm64`, :shrug:

MasterOdin replied:
Alright, looks like there were a few bugs, and that having squashed them, it's now that `popsql/prestodb-sandbox:0.281` only supports `linux/amd64` but that `0.282+` supports that and `linux/arm64`. I'm really not sure what my local state was last week, but retested everything from scratch and it looks good. :+1:

tagomoris commented:
@MasterOdin CIs are failing. Do you have any idea why they're failing? Is it from any differences between images or other reasons?

MasterOdin commented:
Sorry about the number of commits. The PR should now be ready. I had reworked how `ghcr.io/popsql/prestodb-sandbox` images were published to get `linux/arm64` images built, but had introduced a bug in the Dockerfile that rendered the images broken. I've fixed that, and also updated the version used to test to one that has both `linux/amd64` and `linux/arm64` available (this is for 0.282+).

Should be ready to merge now.

tagomoris commented:
Merged. Thank you for the replacement!
